src/base/memory_monitor_sr.cpp

```diff
@@ -41,23 +41,56 @@ namespace cubmem
 
   memory_monitor::memory_monitor (const char *server_name)
     : m_server_name {server_name},
-      m_magic_number {*reinterpret_cast <const int *> ("MMON")}
+      m_magic_number {*reinterpret_cast <const int *> ("MMON")},
+      m_total_mem_usage {0},
+      m_meta_alloc_count {0}
   {}
 
-  size_t memory_monitor::get_allocated_size (const char *ptr)
+  std::string memory_monitor::make_tag_name (const char *file, const int line)
+  {
+    std::string filecopy (file);
+#if defined(WINDOWS)
+    std::string target (""); // not supported
+    assert (false);
+#else
+    std::string target ("/src/");
+#endif // !WINDOWS
+
+    // Find the last occurrence of "src" in the path
+    size_t pos = filecopy.rfind (target);
+#if !defined (NDEBUG)
+    size_t lpos = filecopy.find (target);
+    assert (pos == lpos);
+#endif // !NDEBUG
+
+    if (pos != std::string::npos)
```

Comment:
`make_tag_name()` can get two types of parameter `file` because pre-defined macro `__FILE__` can give two types of return.
1. /home/rudii/cubrid/src/.../filename.cpp
2. filename.cpp

So case 1 touches `if (pos != std::string::npos)` to make substring `.../filename.cpp` and case 2 passes the condition. In case 2, the function returns just `filename.cpp:line`.